index.html

```diff
@@ -1,111 +1,579 @@
+              target="_blank"
+              rel="noopener noreferrer"
+              >Twitter</a
+            >
+            <a
+              href="https://www.facebook.com/yourpage"
+              target="_blank"
+              rel="noopener noreferrer"
+              >Facebook</a
+            >
+            <a
+              href="https://www.Telegram.com/yourpage"
+              target="_blank"
+              rel="noopener noreferrer"
+              >Telegram</a
+            >
+            <a
+              href="https://www.LinkedIn.com/yourpage"
+              target="_blank"
+              rel="noopener noreferrer"
+              >LinkedIn</a
+            >
+            <a
+              href="https://www.Instagram.com/yourpage"
+              target="_blank"
+              rel="noopener noreferrer"
+              >Instagram</a
+            >
+          </div>
+        </div>
+        <div class="team-item">
+          <img src="#" alt="text" />
+          <h3>Jason Smith</h3>
+          <p>Co-Founder</p>
+          <div class="team-list-social">
+            <a
+              href="https://www.Twitter.com/yourpage"
+              target="_blank"
+              rel="noopener noreferrer"
+              >Twitter</a
+            >
+            <a
+              href="https://www.facebook.com/yourpage"
+              target="_blank"
+              rel="noopener noreferrer"
+              >Facebook</a
+            >
+            <a
+              href="https://www.Telegram.com/yourpage"
+              target="_blank"
+              rel="noopener noreferrer"
+              >Telegram</a
+            >
+            <a
+              href="https://www.LinkedIn.com/yourpage"
+              target="_blank"
+              rel="noopener noreferrer"
+              >LinkedIn</a
+            >
+            <a
+              href="https://www.Instagram.com/yourpage"
+              target="_blank"
+              rel="noopener noreferrer"
+              >Instagram</a
+            >
           </div>
         </div>
-      </template>
-      <!-- <script src="script.js"></script> --> <!-- uncomment script --> 
-</body>
+      </section>
+      <section class="faqs">
+        <h2>FAQs</h2>
+        <div class="details">
+          <details>
+            <summary>What are the NTFs?</summary>
+            <p>
+              Urna vitae erat et lacus, consectetur ac nulla vestibulum
+              lobortic. Nula dapibus urna volutpat vestibulum lobortic. Nulla
+              dapibus urna volutpat venenatis, risus faucibus.
+            </p>
+          </details>
+          <details>
+            <summary>How do i get NTFs?</summary>
+          </details>
+          <details>
+            <summary>How can we buy your NTFs?</summary>
+          </details>
+          <details>
+            <summary>Who are the team behind the project?</summary>
+          </details>
+        </div>
+      </section>
+    </main>
+    <footer>
+      <div class="footer">
```

Comment:
It's better to use 'footer-top' class for this element. Class 'footer' can be added to the parent element